mysite/ct/middleware.py

```diff
@@ -16,5 +20,10 @@ def process_exception(self, request, exception):
                                                     kwargs={'user_id': user_id}))
             else:
                 return HttpResponseRedirect(reverse('ct:home'))
+        elif hasattr(smtplib, exception.__class__.__name__):
+            template = loader.get_template('lti/error.html')
+            return HttpResponse(template.render(RequestContext(request,
```

Comment:
It is overhead for this ))
Lets rewrite as:

    return render_to_response(
            'lti/error.html',
            {'message': 'Something goes wrong with email sending. Please try again later.'},
            RequestContext(request)
        )